docs/tutorials/working-with-containers.md

```diff
@@ -119,6 +119,82 @@ You can then access the container via the browser here: [http://localhost:8000/]
 ```
 netsh interface portproxy add v4tov4 listenport=8080 listenaddress=0.0.0.0 connectport=8080 connectaddress=localhost
 ```
+### Exposing the port of a running container
+
+If you forgot to expose the port as part of the `run` command, you can follow the steps below to start a proxy container that forwards traffic to the original container. This hack helps you avoid restarting the container and is especially useful when dealing with containerized services with longer startup times. With full disclosure, this hack is based on the suggestions in this [stackoverflow discussion](https://stackoverflow.com/questions/19897743/exposing-a-port-on-a-live-docker-container) and this [blog post](https://iximiuz.com/en/posts/docker-publish-port-of-running-container/).
+
+1. Let's say you ran a container with out publishing the port (by mistake).
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl run -d --name rd-nginx nginx
+```
+  </TabItem>
+  <TabItem value="docker" default>
+
+```
+docker run -d --name rd-nginx nginx
+```
+  </TabItem>
+</Tabs>
+
+2. Set port variables to be used in the subsequent commands.
+
+```
+# Powershell
+$HOST_PORT=8080
+$CONTAINER_PORT=80
+
+# Bash
+export HOST_PORT=8080
+export CONTAINER_PORT=80
+```
+
```

Comment:
Perhaps we need to add a note for users if they choose to set a priviliged port below 1024 that will then need to be exposed, maybe something like this right before step 3:

:::note
If you choose to set a priviliged port, most Linux distributions (e.g. Ubuntu 20.04) do not allow non-root users to listen on TCP and UDP ports below 1024. To listen to privileged ports on the local host, please run the following command on macOS/Linux:
```
sudo sysctl -w net.ipv4.ip_unprivileged_port_start=80
```
To preserve this change across reboots as a custom kernel parameter setting, add the same command inside your `/etc/sysctl.conf` file.
:::